src/test/kotlin/coverosR3z/timerecording/EnteringTimeBDD.kt

```diff
@@ -87,11 +90,24 @@ class EnteringTimeBDD {
     }
 
     private fun `given I have worked 6 hours on project "A" on Monday with a lot of notes`(): Triple<TimeRecordingUtilities, TimeEntryPreDatabase, RecordTimeResult> {
-        val tru = createTimeRecordingUtility()
-        val newProject: Project = tru.createProject(DEFAULT_PROJECT_NAME)
-        val newEmployee : Employee = tru.createEmployee(DEFAULT_EMPLOYEE_NAME)
+        val pmd = PureMemoryDatabase()
+        val authPersistence = AuthenticationPersistence(pmd)
+        val au = AuthenticationUtilities(authPersistence, currentUserAccessor)
+
+        val tru = TimeRecordingUtilities(TimeEntryPersistence(pmd), currentUserAccessor)
+        val alice = tru.createEmployee(EmployeeName("Alice"))
+
+        au.register("alice", DEFAULT_PASSWORD, alice.id)
```

Comment:
Why don't the names match AAAAAAAHHHHHHHH

Reply:
Since au.register works with users and employees are humans, it's fine and perhaps encouraged for their names to be different. Maybe the inhuman name should be more explicitly username-y